compiler/luci/pass/include/luci/Pass/TransformHardSwishToRelu6Pass.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2023 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __LUCI_TRANSFORM_HARDSWISH_TO_RELU6_PASS_H__
+#define __LUCI_TRANSFORM_HARDSWISH_TO_RELU6_PASS_H__
+
+#include <logo/Pass.h>
+
+namespace luci
+{
+
+/**
+ * @brief  Class to transform HardSwish to Add, Mul and Relu6
+ */
+struct TransformHardSwishToRelu6Pass final : public logo::Pass
```

Comment:
how about `DecomposeHardSwishPass`?

Reply:
Good. I will change the naming.

The `DecomposeHardSwishPass` name seems more accurate.
I was hesitant to use this name because it doesn't seem to have been used before for other passes.

Reply:
Applied in commit: 1acfd3c